src/main/java/org/openrewrite/java/testing/assertj/AdoptAssertJDurationAssertions.java

```diff
@@ -27,4 +57,53 @@ public String getDisplayName() {
     public String getDescription() {
         return "Adopt AssertJ DurationAssert assertions for more expressive messages.";
     }
+
+    @Override
+    public TreeVisitor<?, ExecutionContext> getVisitor() {
+        return new AdoptAssertJDurationAssertionsVisitor();
```

Comment:
Can we add a precondtion here on using either `org.assertj.core.api.DurationAssert has*(..)` or `org.assertj.core.api.AbstractLongAssert isEqualTo(..)`? That could help prevent the `stream().anyMatch` from being evaluated for each method invocation.

Reply:
Even a matcher on `org.assertj.core.api.Assertions assertThat(..)` could limit execution quite a bit already, but more specific is better I think